kafka-client/src/test/java/dev/responsive/kafka/integration/ResponsiveKeyValueStoreRestoreIntegrationTest.java

```diff
@@ -362,35 +357,20 @@ private RemoteKVTable<?> remoteKVTable(
       final TopicPartition changelog
   ) throws InterruptedException, TimeoutException {
     final RemoteKVTable<?> table;
-    if (EXTENSION.backend == StorageBackend.CASSANDRA) {
+
+    if (type == KVSchema.FACT) {
```

Comment:
it looks like we're removing the ability to test with cassandra and instead tying that to the KVSchema type? Am I reading that correctly?

Reply:
Just for this specific test (sorry I meant to leave a comment to clarify before you read this)

My thought was, since we still have at least one customer running Cassandra fact stores then it makes sense to keep some tests parametrized to run that. But no one is running Cassandra KV stores, we would recommend Mongo for that, so we might as well change this to run the KV schema variation against mongo.

Is the method of parametrizing confusing though? I can change it so that the storage backend is the parameter and we determine the schema type based on that.